src/translations/strings.yaml

```diff
@@ -706,7 +706,7 @@ inAppAuth:
     value: "To finish setting up your Mozilla account, create a password in your account settings:"
     comment: User has tried to log in using an SSO-linked account that does not have a password. The user needs to log in to their Mozilla Account to set a password before proceeding.
   ssoInstructionLine2:
```

Comment:
You need to update the string ID here.
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Reply:
Yes, that's the only way to make sure we have the string retranslated. Maybe adding `a` (`ssoInstructionLine2a`) works a bit better than adding `-1`?

Reply:
Thanks! After discussing with the team I ended up changing it to `ssoInstructionLine2-2` because this is the convention we are using elsewhere.

Reply:
Actually, turns out hyphens are not allowed by QML, so I have taken your suggestion.